linked_pages.js

```diff
@@ -0,0 +1,24 @@
+document.querySelector(".instruction_button").addEventListener('click', function () {
+    document.querySelector('.instructions-container').classList.remove('no_show')
+})
+
+
+document.querySelector(".instruction_button").addEventListener('click', function () {
```

Comment:
all these queries could be stored in variables as you're querying more than once